tests/resources/test_project.py

```diff
@@ -439,12 +708,17 @@ def test_delete_project(collection, session):
     assert expected_call == session.last_call
 
 
-def test_update_project(collection, project):
+def test_update_project(collection: ProjectCollection, project):
     project.name = "updated name"
     with pytest.raises(NotImplementedError):
         collection.update(project)
```

Comment:
One of the requirements listed in the ticket is that the Project Collection `update` method should now be able to update both the name and description. Did I miss something here?

Reply:
Let's do this in a followup and check in with product that we actually need updates of team names/descriptions in the sdk

Reply:
Sounds great. Can you link the follow-up ticket to PLA-8395?

Reply:
let's hold off until we get a request from product, I'm not convinced there's a use case for updating team names in the sdk

Reply:
Okay, sounds good.